src/mmw/js/src/modeling/views.js

```diff
@@ -514,14 +515,6 @@ var ToolbarTabContentView = Marionette.CompositeView.extend({
         var modificationsColl = this.model.get('modifications'),
             controlName = model.get('name');
         return modificationsColl.findWhere({ name: controlName });
-    },
-
-    getTemplate: function() {
-        if (this.model.get('is_current_conditions')) {
-            return currentConditionsToolbarTabContentTmpl;
-        } else {
-            return scenarioToolbarTabContentTmpl;
-        }
```

Comment:
Glad to get rid of this, you may want to also delete the "current conditions" template as well.